include/zephyr/usb/class/usbd_hid.h

```diff
@@ -103,6 +103,12 @@ struct hid_device_ops {
 	 */
 	void (*iface_ready)(const struct device *dev, const bool ready);
 
+	/**
+	 * This callback is called when the HID interface is suspended by the
+	 * host. This callback is optional.
+	 */
+	void (*iface_suspended)(const struct device *dev, const bool suspended);
+
```

Comment:
An interface cannot be suspended, and there is nothing like that in the HID specification. To get notifications when a USB device is suspended, please use USBD_MSG_SUSPEND and USBD_MSG_RESUME. There is no reason to have additional callbacks in each function/class implementation.